lib/countly.js

```diff
@@ -1595,16 +1615,20 @@
 
     //insert request to queue
     function toRequestQueue(request) {
-        //ignore bots
-        if (Countly.ignore_visitor) {
-            return;
-        }
+        
+        // disable this controls if test_mode is active
+        if (!Countly.test_mode) {
+            //ignore bots
+            if (Countly.ignore_visitor) {
```

Comment:
Maybe we just need one single check here
```
if(!Countly.ignore_visitor || Countly.test_mode)
```

and nothing more to checking test mode really needed?